askama_derive/src/input.rs

```diff
@@ -118,8 +118,17 @@ impl TemplateInput<'_> {
             let mut nested = vec![parsed.nodes()];
             while let Some(nodes) = nested.pop() {
                 for n in nodes {
+                    let mut add_to_check = |path: PathBuf| -> Result<(), CompileError> {
+                        if !map.contains_key(&path) {
+                            map.insert(path.clone(), Parsed::default());
+                            let source = get_template_source(&path)?;
+                            check.push((path, source));
+                        }
+                        Ok(())
+                    };
+                    use Node::*;
```

Comment:
Please introduce the `use Node::*` in a separate commit at the start of the PR.

Reply:
done

Reply:
I don't understand, do you mean to put this before the main commit?

Reply:
Yes.

Reply:
redone